plugins/inputs/intel_rdt/intel_rdt.go

```diff
@@ -309,8 +309,8 @@ func (r *IntelRDT) processOutput(cmdReader io.ReadCloser, processesPIDsAssociati
 
 			pids, err := findPIDsInMeasurement(out)
 			if err != nil {
-				r.errorChan <- err
-				break
+				r.Log.Warn("skipping text: %w", err)
```

Comment:
How about:

```go
r.Log.Warn("Error reading PID measurements: ", err)
```

Reply:
I wouldn't mention `error`. It seems to me that this should be fine:
```
r.Log.Warnf("Skipping measurement: %w", err)
```